### PR TITLE
Workspace: Decrease default size

### PR DESCRIPTION
**Pull request by tombettany**
KanoComputing/peldins#2215
The default workspace had zooming applied. Remove this and reset any
installs affected by this.

### CODE REVIEW

**Comment by tombettany** [comment]
@pazdera @convolu Bump

File: debian/postinst [hunk 1]
```diff
@@ -13,6 +13,12 @@ case "$1" in
         # Create Symlink
         ln -s /usr/share/sonic-pi/app/gui/qt/sonic-pi /usr/bin/make-music
 
+        if `dpkg --compare-versions $2 lt 2.9-0`; then
```
Comment (convolu) [inline debian/postinst]: Are backticks the right thing here? I don't think that command prints anything

File: debian/postinst [hunk 2]
```diff
@@ -13,6 +13,12 @@ case "$1" in
...
+            # v2.9-0: New defaults for workspace size so remove old ones
+            find /home -path /home/*/.config/uk.ac.cam.cl/Sonic\ Pi.conf \
```
Comment (convolu) [inline debian/postinst]: Out of curiosity, when is this created? I couldn't find it on my image

File: debian/postinst (hunk 1 above)
Comment (tombettany) [inline debian/postinst]: @convolu Creating `test.sh`:

```bash
#!/bin/bash

if `dpkg --compare-versions $1 lt 2.9-0`; then
    echo "Pass"
else
    echo "Fail"
fi
```

and running `./test.sh 2.8` outputs `Pass` but `./test.sh 2.9` outputs `Fail`.

File: debian/postinst (hunk 2 above)
Comment (tombettany) [inline debian/postinst]: I'm not certain but it definitely was on my image. Had you loaded up Sonic Pi before checking?

File: debian/postinst (hunk 1 above)
Comment (convolu) [inline debian/postinst]: I am a bit puzzled, but I find the test case convincing

File: debian/postinst [hunk 3]
```diff
@@ -13,6 +13,12 @@ case "$1" in
...
+                -exec sed -ri '/^workspace[[:digit::]]+zoom=[[:digit:]]+$/d' {} \;
```
Comment (convolu) [inline debian/postinst]: After `workspace`, I think there is an extra `:` after `digit`. Is that a typo, perhaps?

File: debian/postinst (hunk 2 above)
Comment (convolu) [inline debian/postinst]: Disregard my comment, I was using yesterday's image and for some reason make-music was not installed. It does create the file on first run

**Comment by convolu** [comment]
Other than a remark about a potential typo, it looks ok to me

File: debian/postinst (hunk 3 above)
Comment (tombettany) [inline debian/postinst]: Indeed there is, good spot.

**Comment by tombettany** [comment]
@convolu Fixed the typo.

**Comment by convolu** [comment]
sed command looks ok now, tested on my setup and seems to remove the affected lines. Is there any change it messes around with the ownership of the file?

**Comment by tombettany** [comment]
Hmm, I don't think that doing a `sed` can affect file permissions but I could be wrong - @pazdera @Ealdwulf @skarbat any thoughts on if it could?

**Comment by tombettany** [comment]
[This post](http://unix.stackexchange.com/questions/146052/how-is-the-inplace-modification-of-a-file-being-done#146054) indicates that permissions are preserved.

**Comment by skarbat** [comment]
Hmmm.. I agree with Tom. It would if the file did not exist, in which case the `umask` would take hands, but it's not the case I believe. @convolu

**Comment by convolu** [comment]
Awesome, I just wanted to make sure that we won't have another permission issue. 

I'm happy with it @tombettany, go for it :shipit:

**Pull request merged**